common/lib/transport/connectionmanager.js

```diff
@@ -1,9 +1,8 @@
 var ConnectionManager = (function() {
-	var readCookie = (typeof(Cookie) !== 'undefined' && Cookie.read);
-	var createCookie = (typeof(Cookie) !== 'undefined' && Cookie.create);
-	var eraseCookie = (typeof(Cookie) !== 'undefined' && Cookie.erase);
-	var connectionKeyCookie = 'ably-connection-key';
-	var connectionSerialCookie = 'ably-connection-serial';
+	var getFromSession    = (typeof(SessionStorage) !== 'undefined' && SessionStorage.get);
```

Comment:
> Why not just load it and have noop methods like unsupported browsers?

Personally I would prefer to test for the existence of the interface, and use it if present.